timeline.js

```diff
@@ -94,27 +94,21 @@ class ReaderTimeline extends HTMLElement {
     this.loadMoreBtnWrapper.remove()
     let count = 0
 
-    if (this.sort === 'random') {
-      for await (const note of db.searchNotes({}, { limit: this.limit, sort: this.sort === 'random' ? 0 : (this.sort === 'oldest' ? 1 : -1) })) {
+    const notesToShow = await this.fetchSortedNotes()
+    for (const note of notesToShow) {
+      if (note) {
```

Comment:
Is this extra `if` statement necessary? We shouldn't be yielding null from searchNotes.

Also, since we're doing a `for of` anyway, maybe the contents of `fetchSortedNotes` can be inlined into this function so we can render posts as the db makes them available?